### ISSUE TITLE
Wake up from sleep

### ISSUE THREAD

**Issue by chheiss** [Blackymas/NSPanel_HA_Blueprint#554]
Hi! I would love to be able to wake the panel from sleep mode via Home Assistant. I did it using the printf service call page home, but that does not work anymore with current version...

**Comment by edwardtfn** [comment]
About this "Loading..." thing, can you see any message on Home Assistant logs?

**Comment by chheiss** [comment]
page 1 might work. Still testing.

**Comment by lloydbayley** [comment]
Just tested this out some more on my system and it appears that I can make both models do it.

It seems....that the panel doesn't know it's on screensaver or it doesn't know it has come back to the main page.
I might be wrong, but I've checked the web interface whilst programmatically changing pages in HA.

I can send it to screensaver;

service: esphome.nspanel_eu_blk_send_command_printf
data:
  cmd: page home

and then it won't go back to home OR entitypage01 OR buttonpage02 etc. Just stays blank
The scary thing is, in that state, I can't touch it to wake either.

However, if I send it to the page called qrcode, the screen wakes, shows the QR code I selected and then I can send it to 'home' and all is well (including touch).

Hope that's helpful to the programmers. It was fine in 3.1 and I didn't notice until @chheiss brought it up but it does strange things for me too!

**Comment by edwardtfn** [comment]
So, I think I could duplicate this. With latest code in DEV.
In my case, `page screensaver` works fine, the panel goes to sleep, but when sleeping, when I call `page home` it changes the sensor `Currentpage` to `home` almost instantaneously, but takes about 50 secs to wake up.

**Comment by lloydbayley** [comment]
Well-done! Can replicate here too. I was being far-too impatient! @chheiss, can you replicate this and confirm it wakes after waiting about 50 secs?

**Comment by wokwon** [comment]
Confirmed here, 49 seconds.
TFT: 3.2.1
EspHome: 3.2
Blueprint: 3.2.1

I send this:
```
service: esphome.nspensuite_send_command_printf
data:
  cmd: page home
```

and it wakes the screen in 49 seconds.

**Comment by lloydbayley** [comment]
Nothing like confirming a bug and getting it ready for squishing! :)

**Comment by Blackymas** [comment]
A fix will be included in version 3.2.2.

**Comment by Blackymas** [comment]
v3.2.2 is now online
